org.eclipse.lyo.tools.adaptormodel.edit/plugin.properties

```diff
@@ -220,3 +220,7 @@ _UI_MavenProjectConfiguration_groupId_feature = Group Id
 _UI_MavenProjectConfiguration_artifactId_feature = Artifact Id
 _UI_MavenProjectConfiguration_version_feature = Version
 _UI_MavenServerConfiguration_jettyPort_feature = Jetty Port
+_UI_ResourceProperty_rdfCollectionRepresentation_feature = Rdf Collection Representation
+_UI_ResourcePropertyRdfCollectionRepresentation_n_a_literal = n_a
```

Comment:
how about 'Default' or 'Default (repeated predicate)' or 'Default (repeated property)'

Reply:
you mean instead of "n_a", maybe "none" makes more sense. It is not about default. It's about saying there is no RDF Collection.

Reply:
No, here I mean how do I want to serialise a property with ..n cardinality:

- the Default way (just repeat the property n times for a Set with n items)
- use RDF Collection (List)
- use RDF Seq Container
- use RDF Bag Container

Reply:
I see your point.
Let me think about it.